test/unit/math/opencl/kernel_generator/transpose_test.cpp

```diff
@@ -0,0 +1,176 @@
+#ifdef STAN_OPENCL
+#include <stan/math/prim/fun/Eigen.hpp>
+#include <stan/math/opencl/kernel_generator.hpp>
+#include <stan/math/opencl/matrix_cl.hpp>
+#include <stan/math/opencl/copy.hpp>
+#include <test/unit/math/opencl/kernel_generator/reference_kernel.hpp>
+#include <stan/math.hpp>
+#include <gtest/gtest.h>
+#include <string>
+
+using Eigen::MatrixXd;
+using stan::math::matrix_cl;
```

Comment:
I think its usually good practice to put these inside of the tests than have them floating in global

Reply:
fixed